src/components/standup/index.tsx

```diff
@@ -18,6 +23,49 @@ const StandUpContainer: FC = () => {
     const standupDates = userStandupdata?.data?.map((element) => element.date);
     const totalMissedUpdates = getTotalMissedUpdates(standupDates || []);
     const currentDate = getCurrentDate(STANDUPTIME);
+    const result = standupDates?.reduce(
+        (max, current) => (current > max ? current : max),
+        standupDates[0]
+    );
+    const getDateFromTimeStamp = moment(result).format('DD-MM-YYYY');
+    const getCurrentDateResult = moment().format('DD-MM-YYYY');
+    const currentDateResult = getCurrentDateResult.split('-')[0];
+    const dateResult = getDateFromTimeStamp.split('-')[0];
+    const now = moment();
+    const hour = now.hours();
+
+    useEffect(() => {
+        if (isLoading) {
+            return;
+        }
+        if (currentDateResult === dateResult && standupDates) {
+            setIsFormVisible(false);
+        } else if (hour >= 6 && currentDateResult !== dateResult) {
+            setIsFormVisible(true);
+        } else {
+            setIsFormVisible(true);
+        }
+    }, [standupDates]);
+
+    const mainForm = () => {
+        if (isLoading) {
+            return <Loader />;
+        } else {
+            if (isFormVisible) {
+                return (
+                    <FormInputComponent setIsFormVisible={setIsFormVisible} />
+                );
+            } else {
+                return (
+                    <p className={styles.formFilledMessage}>
+                        Your standup for the day has already been submitted,
+                        please fill out the form tomorrow after 6:00 a.m.
+                    </p>
+                );
+            }
+        }
+    };
+
```

Comment:
Please increase the test coverage of uncover line

Reply:
Test coverage is not getting increased by 90%.

Reply:
If possible try this uncover line to be cover